### PR TITLE
Adding changes to MySQL Packages and pyZypp

### PR DESCRIPTION
**Pull request by dubb-b**
Adding changes to MySQL Packages and pyZypp, also adding py3 support for them.

The zypper search has them listed as the following now:

python2-PyMySQL
python2-zypp-plugin

python3-PyMySQL
python3-zypp-plugin

### CODE REVIEW

File: python/mysqldb.sls [hunk 1]
```diff
@@ -6,14 +6,18 @@
...
 {% elif grains['os_family'] == 'FreeBSD' %}
   {% set mysqldb = 'py27-MySQLdb' %}
 {% else %}
   {% set mysqldb = 'python-mysqldb' %}
 {% endif %}
 
+{% if- pillar.get('py3', False) and grains['os_family'] == 'Suse' %}
+  {% set mysqldb = 'python3-PyMySQL' %}
+{% else %}
+  {% set mysqldb = 'python2-PyMySQL' %}
```
Comment (rallytime) [inline python/mysqldb.sls]: This is going to override all of the work done about in the if/elif block. Probably not what you intended here. :)

Comment (dubb-b) [reply python/mysqldb.sls]: @rallytime this has been updated, thanks for catching that.

**Comment by dubb-b** [comment]
@Ch3LL we can hold off on this until we get the container working.

**Comment by Ch3LL** [comment]
sure

**Pull request closed without merging**